zerver/migrations/0143_realm_allow_community_topic_editing.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-09-28 22:56
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def migrate_past_realms_allow_community_topic_editing_false(apps, schema_editor):
+    # type: (StateApps, DatabaseSchemaEditor) -> None
+    Realm = apps.get_model("zerver", "Realm")
+    realms = Realm.objects.all()
+    for realm in realms:
+        realm.allow_community_topic_editing = False
+        realm.save()
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0142_userprofile_translate_emoticons'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='realm',
+            name='allow_community_topic_editing',
+            field=models.BooleanField(default=True),
+        ),
+        migrations.RunPython(migrate_past_realms_allow_community_topic_editing_false),
```

Comment:
You should add a `reverse_code` here that's noop (see how our other `RunPython`s do it) so one can undo this migration.